downloader/src/main/kotlin/vcs/Git.kt

```diff
@@ -59,30 +58,32 @@ const val GIT_HISTORY_DEPTH = 50
 class Git : VersionControlSystem(), CommandLineTool {
     companion object {
         init {
-            // Configure JGit to connect to the SSH-Agent if it is available.
             val sessionFactory = object : JschConfigSessionFactory() {
-                override fun configure(hc: OpenSshConfig.Host, session: Session) {
-                    session.setConfig("StrictHostKeyChecking", "no")
-                }
+                @Suppress("EmptyFunctionBlock")
+                override fun configure(hc: OpenSshConfig.Host, session: Session) {}
+
+                override fun configureJSch(jsch: JSch) {
+                    // The default JSch instance uses identity files named "identity", "id_rsa" or "id_dsa" from the
+                    // current user's ".ssh" directory, see https://www.codeaffine.com/2014/12/09/jgit-authentication/.
```

Comment:
I assume this comment is there to explain the behavior if no SSH-Agent is available? If so, maybe merge it with the SSH-Agent comment below.

Reply:
The comment is there to explain which kind of authentication JGit configures JSch for by default. I've nevertheless merged with the comment below.